### ISSUE TITLE
About the comparison experiment with GL

### ISSUE THREAD

**Issue by Mistariano** [NVIDIA/partialconv#10]
Hi,
Thanks a lot for your brilliant work on inpainting with partial conv. It helped me a lot. :)

I'm a bit puzzled at your comparison experiment with Globally and Locally Consistent Image Completion. According to the paper, you are using the pretrained model of GL, which was trained on Place2 dataset, and you compared the results on both Place2 and ImageNet (figure 5 & 6). 
I just wonder why you decided to test it on ImageNet? Do you mean that the images in Place2 and ImageNet are similiar so we can just use the latter to validate models trained on the former?

Looking forward to your reply. Thanks again!

**Comment by liuguilin1225** [comment]
@Mistariano Yes. We have mentioned that we use the Place2 model released by "Globally and Locally Consistent Image Completion" paper for the comparisons.
"Note that the released model for [10] was trained only on Places2, which we use for all evaluations."
We did that partly because the training code is not released by the author and only the pretrained model on Place2 has been released.

**Comment by Mistariano** [comment]
Appreciate, it helped me a lot.
Thus, before comparing the test results of yours and GL, you resized the output of GL from 256x256 to 512x512, so that they have the same size, is it right?
BTW, would you mind giving me some advice for training? I'm eager to know which batch-size you are using, and after how many epochs/iterations your model can converge.

**Comment by liuguilin1225** [comment]
@Mistariano For GL, I think we did 512x512 directly; for the GntIpt, we do 256x256 then upsample the hole region. 
For batch size, it depends on your GPU memory; we used 5 or 6. We trained in on V100 for about 10 days.
